tests/util/starknet.ts

```diff
@@ -248,7 +249,7 @@ export function transfer(
     version: 1, // version of the transaction
     signature: [], // leave empty for now, will be filled in when signing the transaction
     sender_address: contractAddress, // address of the sender contract
-    nonce: nonce || 0, // nonce of the transaction
+    nonce: numberToU832Bytes(nonce ? nonce : 0), // nonce of the transaction
```

Comment:
If you are wondering about this function, when running the typescript tests for the extrinsic, after changing nonce to Felt252Wrapper, it was expecting it to be in a 32 bytes format, hence this utility was added which converts the nonce to a 32 bytes uint format.